src/hooks/useAdvancedVoice.test.ts

```
@@ -0,0 +1,359 @@
+    expect(result.current.config.persona).toBe('oracle');
+    expect(result.current.config.rate).toBe(1.2);
+  });
+
+  it('detects browser support correctly', () => {
+    const { result } = renderHook(() => useAdvancedVoice());
+    expect(result.current.isSupported).toBe(true);
+
+    // Test without speechSynthesis
+    const originalSpeechSynthesis = global.speechSynthesis;
+    // @ts-ignore
+    delete global.speechSynthesis;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)